recipes/rnaz/build.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-./configure --prefix=$PREFIX --datadir=$PREFIX/share
+./configure CFLAGS="-std=gnu89 -g -O2" --prefix=$PREFIX --datadir=$PREFIX/share
 make
 make install
-cp $PREFIX/share/RNAz/perl/*.pl $PREFIX
```

Comment:
we could symlink the perl scripts that are needed into $PREFIX/bin isn't it?

Reply:
but then we also need to make the RNAz.pm available in the $PERL5LIB variable, or copy that as well to $PREFIX/bin, or have perl in a pinned version as dependency so I know what $PREFIX/lib/perl5/site-perl version to use